backend/api/battles/serializers.py

```diff
@@ -66,30 +71,30 @@ def validate(self, attrs):
 
 
 class DetailBattleSerializer(serializers.ModelSerializer):
-    creator_team = serializers.SerializerMethodField()
-    opponent_team = serializers.SerializerMethodField()
+    creatorTeam = serializers.SerializerMethodField()
+    opponentTeam = serializers.SerializerMethodField()
```

Comment:
Hey I was in doubt about that, I'm partial to using camelCase on frontend but eslint raises an error and I was questioning if I should disable the error. I guess I'll disable it to avoid confusion